mitodl/open-discussions
# Added redirect to MM on login

#### What are the relevant tickets?
Closes #673 

#### What's this PR do?
Adds a login step that prompts a user to login with MM if that's their only authentication method

#### How should this be manually tested?
- Set `SOCIAL_AUTH_MICROMASTERS_LOGIN_URL=http://mm.dev.local:8079/discussions/`
- Ensure your user only has a "micromasters" social auth 
- Go to `/login` and enter the email for this user. You should see the page and be directed to login in MM.
